services/movie.py

```diff
@@ -0,0 +1,35 @@
+from django.db import models
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list = None,
+        actors_ids: list = None
+) -> models.QuerySet:
+
+    movies = Movie.objects.all()
+    if genres_ids:
+        movies = movies.filter(genre__id__in=genres_ids)

```

Comment:
The filter should use `genres__id__in` instead of `genre__id__in` to match the `ManyToManyField` name `genres` in the `Movie` model.